### PR TITLE
Added delete request type to A

### PR DESCRIPTION
**Pull request by linslin**
Added delete request type to A

### CODE REVIEW

File: src/Client/Utils/A.php [hunk 1]
```diff
@@ -50,4 +50,12 @@ public static function fetchRequest()
...
-}
+
+    /**
+     * @return \Mcustiel\Phiremock\Client\Utils\RequestBuilder
+     */
+    public static function deleteRequest()
+    {
+        return RequestBuilder::create('delete');
+    }
+}
```
Comment (mcustiel) [inline src/Client/Utils/A.php]: Please add a blank line at the end of the file and we are ready to merge. Never saw this one was missing, thank you.

Comment (linslin) [reply src/Client/Utils/A.php]: I will, but why do you need an blank line at the end of the doc?

Comment (linslin) [reply src/Client/Utils/A.php]: Oh yea IC, EOF expectation.

**Comment by mcustiel** [comment]
Tonight I will add a new version with this fix because I need to create the phar file and I can't do it right now. In the meantime you can use dev-master.
Thank you very much.

**Comment by linslin** [comment]
I'm fine. I'll add the delete test later. Thanks m8

**Comment by linslin** [comment]
@mcustiel thanks for this quick release. Now we could reach 100% coverage!

**Pull request merged**